docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -1150,8 +1317,34 @@ into bytes that can be stored in the validator's Radix-Merkle tree.
 
 {% endif %}
 
+Addressing
+----------
+
 By convention, we'll store game data at an address obtained from hashing the
-game name prepended with some constant:
+game name prepended with some constant.
+
+XO data is stored in state using addresses generated from the XO
+family name and the name of the game being stored. In particular, an
+XO address consists of the first 6 characters of the SHA-512 hash of
+the UTF-8 encoding of the string "xo" (which is "5b7349") plus the
+first 64 characters of the SHA-512 hash of the UTF-8 encoding of the
+game name.
+
+For example, the XO address for a game called "mygame" could be
```

Comment:
This is my error (oops), but could you change "mygame" to "my-game"? Then it will match the name in the following example.  Thanks!

Reply:
This is from commit dd5ff64 "Edit & expand text in SDK tutorials; reorganize chapter for better flow". Since that is in your PR it makes sense to change it in yours. When your PR goes into master I will rebase mine.